lib/services/network_service.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+
+import 'package:module_nine/model/photo.dart';
+
+const _url = 'https://jsonplaceholder.typicode.com/photos';
+
+class NetworkService {
+  Future<List<Photo>?> get() async {
+    final response = await http.get(Uri.parse(_url));
+    try {
+      if (response.statusCode != 200) {
```

Comment:
No need to put this check into try block. The main rule for try is to put there only the code that depends on the exception that you want to catch.

Reply:
done